frameworks/helloworld/universe/config.json

```diff
@@ -144,6 +144,11 @@
           "enum": [0, 1],
           "default": 1
         },
+        "enable_backoff": {
```

Comment:
I think we should extend this from a simple boolean flag into a subsection under `service` i.e
```
"task_failure_backoff": {
  "disabled" : true
  "backoff-factor": {...}
  "max-launch-delay": {...}
  "initial-backoff": {...}
}
```
In addition, we should switch the flag from `enabled` to `disabled` such that for now it will be opt-in but after the next n+2 releases, we can flip the `disabled` flag to default to false.